parser.jison

```diff
@@ -72,6 +73,7 @@ expression
     | css_variable { $$ = $1; }
   	| css_value { $$ = $1; }
   	| value { $$ = $1; }
+    | UNKNOW_FUNCTION { $$ = { type: 'Unknow', value: $1 }; }
```

Comment:
If the name is to be kept, it should be "unknow*n* function". But I'm leaning towards normalisation, so maybe just "function".